src/org/etools/j1939_84/bus/j1939/Lookup.java

```diff
@@ -144,6 +146,8 @@ public static String getPartName(int partNumber) {
      *                 the suspect parameter number from the Diagnostic Trouble Code
      * @return     The name as defined by SAE or "Unknown" if it's not defined
      */
+    @SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE", justification = "Null check needs to happen here")
+
     public static String getSpnName(int spn) {
         SpnDefinition spnDef = J1939DaRepository.getInstance().findSpnDefinition(spn);
```

Comment:
this method will always return a non-null value.  The check needs removed and not suppressed.